packages/presentation/lib/utils/widget_display_helper.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:domain/domain.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:presentation/const/app.dart';
+import 'package:presentation/style/dimens.dart';
+
+enum ScreenType { desktop, tablet, phone }
+
+class WidgetDisplayHelper {
+  const WidgetDisplayHelper._();
+
+  static ScreenType getDisplayType(BuildContext context) {
```

Comment:
maybe create late final variable context and create constructor init where you init your context

Reply:
Then you can call WidgetDisplayHelper without context

Reply:
I tried to do so, and initialised context in stream builder while first empty tile comes. But my responsive ui becomes dead after that) seems like I need to use different contexts in different cases, so I will roll back this changes. Or u can advise something else?